### ISSUE TITLE
Bug: `Worksheet.autofit` panics if a cell has a long string

### ISSUE THREAD

**Issue by StefanBRas** [jmcnamara/rust_xlsxwriter#115]
### Current behavior

If you run `Worksheet.autofit()` on a worksheet where at least one cell has a very long string, it will panic.

### Expected behavior

That it wouldn't panic

### Sample code to reproduce

```rust
use rust_xlsxwriter::Workbook;

fn main() {
    let very_long_string = "X".repeat(10000);
    Workbook::new()
        .add_worksheet()
        .write_string(0, 0, very_long_string)
        .unwrap()
        .autofit();
}

```

### Environment

- `rust_xlsxwriter` version: "0.79.2"
- Cargo.toml dependency line for `rust_xlsxwriter`: `rust_xlsxwriter = "0.79.2"`
- rustc version: rustc 1.81.0 (eeb90cda1 2024-09-04)
- Excel version: NA
- OS: Mac
- If using wasm, which method/tool:

### Any other information

It happened while i was PoC'ing exporting some data. I probably don't actually want to autofit that wide a column (and maybe not even include it).  It just happened because I wasn't aware that the string would end up being that large. 

I have no strong opinions about a solution - you could do `saturating_add` but then there's a performance penalty for a case that doesn't really make sense to support. I was just surprised that it could panic, since the doc string didn't mention it and it seemed to safe to run.

**Comment by jmcnamara** [comment]
Thanks for the report. 

I don't see a panic when I run the sample code. I tried with a similar environment to you but with rust 1.80.0 and 1.82.0. Am I missing something?

**Comment by jmcnamara** [comment]
Sorry, my mistake. I see the panic now.

**Comment by jmcnamara** [comment]
I've pushed a fix for this to main. I added an upper limit of 1790 pixels for the autofit width, in line with Excel. That also avoids the integer overflow. Thanks of the report.

**Comment by jmcnamara** [comment]
You can test it by adding the GitHub repo to your `Cargo.toml` file:

```
cargo add --git https://github.com/jmcnamara/rust_xlsxwriter.git rust_xlsxwriter
```

Let me know how you get on and I'll roll it up into a release.

**Comment by StefanBRas** [comment]
I unfortunately already changed too much on my original PoC to reproduce it there, but I tested it with the minimal reproduction case and that works now

Thanks for the quick fix!

**Comment by jmcnamara** [comment]
Fixed in `rust_xlsxwriter v0.79.3`.
